test/puppetlabs/trapperkeeper/services/status/status_service_test.clj

```diff
@@ -174,6 +176,7 @@
       app
       [jetty9-service/jetty9-service
        webrouting-service/webrouting-service
+       scheduler-service/scheduler-service
        status-service]
       (merge status-service-config
         {:web-router-service {:puppetlabs.trapperkeeper.services.status.status-service/status-service "/alternate-status"}})
```

Comment:
Thinking about it more, I'm not sure there is a sane way to test it, since the log message will just be a json string, it'll be hard to call `logged?` with some regex. We could redef the logging function with a promise or something to make it more synchronous, but then we're just testing tk-scheduler, not that the service logs correctly ¯\\\_(ツ)\_/¯